code/game/machinery/vitals_monitor.dm

```diff
@@ -128,11 +231,24 @@
 	var/mob/user = usr
 	if(!istype(user))
 		return
-	
+
 	if(CanPhysicallyInteract(user))
 		beep = !beep
 		to_chat(user, SPAN_NOTICE("You turn the sound on \the [src] [beep ? "on" : "off"]."))
-		
+
+/obj/machinery/vitals_monitor/verb/toggle_alerts()
```

Comment:
Vitals monitor currently has no `attack_hand()` definition, could use that instead of a verb.